collectors/networkviewer.plugin/README.md

```diff
@@ -0,0 +1,85 @@
+# networkviewer.plugin
+
+This plugin uses eBPF to bring connection information from the kernel ring to user ring. 
```

Comment:
I would suggest changing this to say:

'This plugin uses eBPF to collect information about network connections directly from the kernel.'

The 'ring' terminology used for privilege levels is not very widely used outside of talking about low-leve hardware privilege levels (and even then is mostly an x86 thing). Even ignoring that though, the current wording is a bit atypical (it's not particularly consistent with a lot of the rest of the documentation, and it took me a moment to understand exactly what it meant, neither of which are good things for documentation).

Reply:
Thank you, I changed the documentation following your suggestion.